main_test.go

```diff
@@ -226,18 +226,19 @@ func Test_logTaskEnv(t *testing.T) {
 		}
 
 		Convey("dumps the vars it finds", func() {
-			logTaskEnv(taskInfo)
+			logTaskEnv(taskInfo, container.LabelsForTask(taskInfo))
 
 			So(string(output.Bytes()), ShouldContainSubstring, "--------")
 			So(string(output.Bytes()), ShouldContainSubstring, "BOCACCIO=author")
 		})
 
-		Convey("reports when there are none", func() {
+		Convey("always has environment and service name", func() {
 			taskInfo.Container.Docker.Parameters = []*mesos.Parameter{}
 
-			logTaskEnv(taskInfo)
+			logTaskEnv(taskInfo, container.LabelsForTask(taskInfo))
 
-			So(string(output.Bytes()), ShouldContainSubstring, "No Docker environment")
+			So(string(output.Bytes()), ShouldContainSubstring, "SERVICE_NAME=undefined-servicename")
+			So(string(output.Bytes()), ShouldContainSubstring, "ENVIRONMENT_NAME=")
```

Comment:
not sure how this is getting service name  as undefined-servicename, but environment_name as blank? other code is setting up default as dev if it's iunavailable.  mind you, i also thought it wouldn't set up underfined-servicename if no labels there... could you tell me what to expect for the defaults?